src/intentLogic/getLatestReviews.js

```diff
@@ -0,0 +1,86 @@
+const get = require('simple-get-promise').get;
+const asJson = require('simple-get-promise').asJson;
+
+const helpers = require('../helpers');
+const speech = require('../speech').speech;
+const sound = require('../speech').sound;
+const getMoreOffset = require('../helpers').getMoreOffset;
+
+module.exports = function(isNewIntentFlag) {
+    const isNewIntent = isNewIntentFlag !== false;
+
+    const attributes = this.event.session.attributes;
+    const slots = this.event.request.intent.slots;
+
+    attributes.lastIntent = 'GetLatestReviewsIntent';
+
+    if (isNewIntent) attributes.reviewType = slots.review_type.value;
+
+    if (attributes.reviewType) {
+        attributes.moreOffset = getMoreOffset(isNewIntent, attributes.moreOffset);
+
+        const filter = "page="+ ((attributes.moreOffset / helpers.pageSize) + 1)
+                     + "&page-size="+ helpers.pageSize
+                     + "&tag=tone/reviews,"+ getTagType(attributes.reviewType)
+                     + "&show-fields=standfirst,byline,headline&show-blocks=all";
+
+        const capiQuery = helpers.capiQuery('search', filter);
+        get(capiQuery)
+            .then(asJson)
+            .then(json => {
+                if (json.response.results && json.response.results.length > 1) {
+                    attributes.positionalContent = json.response.results.map(review => review.id);
+                    const preamble = getPreamble(isNewIntent, json.response.results.length, attributes.reviewType);
+                    const reviews = json.response.results.map(review => {
+                        return review.fields.headline + sound.transition;
+                    });
+                    const conclusion = getConclusion(json.response.results.length);
+
+                    this.emit(':ask', `${preamble} ${reviews} ${conclusion}`);
+                } else {
+                    this.emit(':tell', speech.reviews.notfound);
+                }
+            })
+            .catch(error => {
+                this.emit(':tell', speech.reviews.notfound);
+            })
+    } else {
+        this.emit(':ask', speech.reviews.clarifyType);
+    }
+}
+
+function getTagType(reviewType) {
```

Comment:
No they're not :/ From the Alexa documentation:

> Note that a custom slot type is not the equivalent of an enumeration. Values outside the list may still be returned if recognized by the spoken language understanding system. Although input to a custom slot type is weighted towards the values in the list, it is not constrained to just the items on the list. Your code still needs to include validation and error checking when using slot values.

[Source](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/alexa-skills-kit-interaction-model-reference#h2_custom_syntax)